engine/src/main/java/org/terasology/engine/subsystem/lwjgl/LwjglGraphics.java

```diff
@@ -254,6 +254,7 @@ private void checkOpenGL() {
                 GLContext.getCapabilities().OpenGL12,
                 GLContext.getCapabilities().OpenGL14,
                 GLContext.getCapabilities().OpenGL15,
+                GLContext.getCapabilities().OpenGL20,
```

Comment:
Yes, being able to select a render would be good. Lots of hurdles to get there, especially with regards to shaders (we would probably need our own shader format that allows for fallback options). It would be particularly good to have the main render using the OpenGL core profile rather than compatibility profile, with a pure compatibility renderer as a backup.